RustFxA/FxAWebViewModel.swift

```diff
@@ -9,17 +9,6 @@ import MozillaAppServices
 import Shared
 import SwiftKeychainWrapper
 
-
-extension URL {
-    var toRequest: URLRequest {
-        return URLRequest(url: self)
-    }
-    /// Figuring out if two urls are the same based on scheme, host and path.
-    func isEquivilent(to url: URL) -> Bool {
-        return self.scheme == url.scheme && self.host == url.host && self.path == url.path
-    }
-}
-
```

Comment:
Note on URL extensions:
- already extensive, disagree with adding more for single-use 
- we have an URL extension class, how would someone know to look here?
- spelling is wrong
- equivalency is not something we want to add broadly, is a complicated topic, there is a specific form of check used here we can inline